packages/components/src/InputSelect/Component.purs

```diff
@@ -0,0 +1,99 @@
+module KSF.InputSelect.Component where
```

Comment:
You can avoid the old convention of creating a folder for each component and put the code directly in a file called `InputSelect.purs`

Reply:
And maybe we could think of a better name, cause `InputSelect` remembers me of the HTML `select` tag and this might generate confusion.

Reply:
If we want to separate the "selectable" input types into their own module, it would make sense to place it under `KSF.InputField`. So it would be named something like `KSF.InputField.Selectable`. This way we can still treat text inputs and boolean inputs as `InputFields` on the outside (if we want), and the type `data InputType = Text | Password | Email | Radio | Checkbox` would stay coherent.

Reply:
`KSF.InputField.Selectable` makes a lot of sense. Should this then be grouped under the InputField folder?

Reply:
Preferably yes.

Reply:
continued here https://github.com/KSF-Media/affresco/pull/417#discussion_r487782364